tests/din/session_setup.cpp

```diff
@@ -222,4 +208,4 @@ SCENARIO("Encode and decode DIN session setup message") {
     //            REQUIRE(session_setup_res.DateTimeNow == 1675074582);
     //        }
     //    }
-}
+}
```

Comment:
Noted!

Yes, I gave the DIN test issue table an ISO stream instead of a DIN for the SessionSetupRes message. But Tobias and I didn't want to delete this code. I have to generate the correct din exi stream, then we can uncomment the code again.